src/useDCPWorker.tsx

```diff
@@ -227,20 +166,26 @@ const defaultWorkerStats: IDefaultWorkerStats = {
   slices: 0,
   credits: new BigNumber(0),
   computeTime: 0,
-  options: { paymentAddress: null, maxWorkingSandboxes: 0 },
 };
 
+enum WorkerStatsActions
+{
+  ADD_COMPUTE_TIME = 'ADD_COMPUTE_TIME',
+  ADD_SLICE = 'ADD_SLICE',
+  ADD_CREDITS = 'ADD_CREDITS'
+}
+
 const workerStatsReducer = (
   state: IDefaultWorkerStats,
   action: { type: string; data: any },
 ) => {
-  let updatedStats = { ...state };
+  const updatedStats = { ...state };
 
-  if (action.type === 'INCREMENT_COMPUTE_TIME') {
+  if (action.type === WorkerStatsActions.ADD_COMPUTE_TIME) {
     updatedStats.computeTime += action.data;
-  } else if (action.type === 'INCREMENT_SLICES') {
+  } else if (action.type === WorkerStatsActions.ADD_SLICE) {
     updatedStats.slices++;
-  } else if (action.type === 'INCREMENT_CREDITS') {
+  } else if (action.type === WorkerStatsActions.ADD_CREDITS) {
     updatedStats.credits = updatedStats.credits.plus(action.data);
   }
```

Comment:
**nitpick:** If other braces will be changed to Allman style, these braces should be changed as well.

Reply:
fixed to use switch case instead